easybuild/easyconfigs/n/nodejs/nodejs-12.16.1-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'ConfigureMake'
+
+name = 'nodejs'
+version = '12.16.1'  # 12.16.1 was LTS on 2020-05-22
```

Comment:
Did you mean `2020-05-22`, or do you have a time travel machine? ;-)

Reply:
Thanks 2020-03-22 ofc :) Apparently wishing to be May already.

Reply:
Don't we all... ;-)